api/models/task.py

```diff
@@ -9,4 +18,7 @@ class Task(db.Model):
         UUIDType(binary=True), ForeignKey("current_session.user_id"), nullable=False
     )
     title = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    completed = Column(Boolean, nullable=False)
     created = Column(DateTime, server_default=func.now())
+    duration = Column(Time, nullable=False)
```

Comment:
duration should be nullable - it will save us from implementing this feature now and in the future we just won't schedule this tasks ourselves

Reply:
we are not planning on using it now, but potentially in the future, so we can probably leave this as is to not do another schema change (+ db migration)